setup_bitswap.go

```diff
@@ -21,7 +23,16 @@ import (
 
 func setupBitswapExchange(ctx context.Context, cfg Config, h host.Host, cr routing.ContentRouting, bstore blockstore.Blockstore) exchange.Interface {
 	bsctx := metri.CtxScope(ctx, "ipfs_bitswap")
-	bn := bsnet.NewFromIpfsHost(h, cr)
+
+	bn := bsnet.NewFromIpfsHost(h)
+
+	// Custom query manager with the content router and the host
+	// and our custom options to overwrite the default.
+	pqm, err := providerquerymanager.New(ctx, h, cr, providerquerymanager.WithMaxInProcessRequests(100))
+	if err != nil {
+		panic(err)
+	}
+	pqm.Startup()
```

Comment:
Unless there is a need to start at some later time, or some modes of operation need a `ProviderQueryQanager` that has not been started, then I think `New` should also perform the startup.

Reply:
Doing so prevents problems where calling `Startup` is missed.

Reply:
https://github.com/ipfs/boxo/pull/734